railties/test/test_unit/test_parser_test.rb

```diff
@@ -38,6 +38,19 @@ def test_oneline; assert true; end
     assert true
     assert_not false
   }
+
+  # Emulate minitest/spec. Check that extensions can provide aliases
```

Comment:
Can we use something else to test this? We really don't want to make like we support minitest/spec or any syntax that looks like rspec.

Reply:
Yeah, sure. I made it look more generic